qa/suites/fs/upgrade/upgraded_client/branch/n-1.yaml

```diff
@@ -0,0 +1,2 @@
+teuthology:
```

Comment:
why leave out quincy (yaml) here? @dparmar18

Reply:
quincy is n-3

Reply:
and i've just added n-1|n-2 + restructured the yamls . `upgraded_client` has not quincy start point, so i didn't make it. if you want i can add it

Reply:
We can add it, but my question is why was it missing till now (isn't there a need to test it?). This is upgraded_client, so my only explanation is that this was added to test out upgrades for a particular issue that wasn't totally required to be tested with quincy.

Reply:
so let me confirm this again, we don't want to do from r|s->main but q|r->squid for `upgraded_client` sub-suite, correct?

Reply:
It has been this way when it started, but I think we can change that to test client running branch sha with reef/squid MDS. @lxbsz @batrick wdyt?

Reply:
IMO we should keep `quincy --> main`, `reef --> main` and `squid --> main` always. And when there has new release we should add new test for it and shouldn't delete the existing ones.

This may help detect some potential bugs as we fixed `new getvxattr op` one.

Reply:
we cant do quincy to main, it would be more than 2 major releases.

Reply:
We should have testing for branch sha client and squid/reef MDS testing. We rely on testing older releases for this sub-suite in their respective release branch testing? Does that sound reasonable?

Reply:
@vshankar didn't quite get `branch sha client`

Reply:
Right now `main` is in a halfway point between releases. It's still basically `squid` so testing from `quincy` is fine.

(Let's discuss this PR's direction more tomorrow.)